dcicutils/ff_utils.py

```diff
@@ -1145,8 +1154,8 @@ def unified_authentication(auth=None, ff_env=None):
     elif isinstance(auth, tuple) and len(auth) == 2:
         use_auth = auth
     if not use_auth:
-        raise Exception("Must provide a valid authorization key or ff "
-                        "environment. You gave: %s (key), %s (ff_env)" % (auth, ff_env))
+        raise ValueError("Must provide a valid authorization key or ff environment."
```

Comment:
Might consider adding some custom exceptions in the library? Definitely an improvement over `Exception` though.